### DISCUSSION TITLE
compile be module at  branch 0.14 failed on apachedoris/doris-dev:build-env-1.3 image

### DISCUSSION THREAD

**Discussion by James601232** [apache/doris#7268] [RFC]
[208/678] Building CXX object src/http/CMakeFiles/Webserver.dir/action/mini_load.cpp.o
In file included from /var/local/thirdparty/installed/include/boost/bind.hpp:30,
                 from ../src/runtime/client_cache.h:21,
                 from ../src/http/action/mini_load.cpp:51:
/var/local/thirdparty/installed/include/boost/config/pragma_message.hpp:24:34: note: '#pragma message: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.'
   24 | # define BOOST_PRAGMA_MESSAGE(x) _Pragma(BOOST_STRINGIZE(message(x)))
      |                                  ^~~~~~~
/var/local/thirdparty/installed/include/boost/config/pragma_message.hpp:24:34: note: in definition of macro 'BOOST_PRAGMA_MESSAGE'
   24 | # define BOOST_PRAGMA_MESSAGE(x) _Pragma(BOOST_STRINGIZE(message(x)))
      |                                  ^~~~~~~
ninja: build stopped: subcommand failed.

**Comment by morningman** [comment]
See
http://doris.incubator.apache.org/master/zh-CN/installing/compilation.html#%E4%BD%BF%E7%94%A8-docker-%E5%BC%80%E5%8F%91%E9%95%9C%E5%83%8F%E7%BC%96%E8%AF%91-%E6%8E%A8%E8%8D%90

to get the right docker image. 0.14.0 should use 1.2
